Readme.md

```diff
@@ -22,6 +22,51 @@ Documentation
 
 1. [Development Process][docs.1.development-process]
 
+Prerequisites
+-------------
+
+Praefectus is a .NET Core 3.1 application, that is published in a
+[self-contained deployment mode][dotnet-publish.self-contained]. It means that
+to run the released binaries, you'll need to install [.NET Core
+prerequisites][dotnet-prerequisites] for your operating system.
+
+For developers, [.NET Core 3.1 SDK][dotnet-download] is required.
+
+Build
+-----
+
+To build the application, run the following command in the terminal:
+
+```console
+$ dotnet build --configuration Release
+```
+
+To publish the application (prepare a distributable copy independent of
+installed runtime), run the following command:
+
+```console
+$ dotnet publish --self-contained true --configuration Release --runtime <RUNTIME_IDENTIFIER> --output publish Praefectus.Console
+```
+
+Here `<RUNTIME_IDENTIFIER>` is a [RID][dotnet-rid] for the target platform.
+Currently used RIDs are:
+- `linux-x64`
+- `osx-x64`
+- `win-x64`
+
+This will create a redistributable set of application files in the `publish`
+directory. `praefectus` (or `praefectus.exe`) binary in a console entry point.
```

Comment:
Should this say "`praefectus … *is* a console entry point"?

Reply:
Yup, thanks.